From db067b9054d87ada6aa133394e65e3af9d75fc08 Mon Sep 17 00:00:00 2001
From: David Teigland <teigland@redhat.com>
Date: Tue, 3 Jan 2023 11:38:33 -0600
Subject: [PATCH] lvresize: fix cryptsetup resize in helper

typo used "cryptresize" as command name

this affects cases where the file system is resized
independently, and then the lvresize command is used
which only needs to resize the crypt device and the LV.

(cherry picked from commit 81acde7ffdf9fbe522ada16f89e429d9f729dc0c)
---
 scripts/lvresize_fs_helper.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/lvresize_fs_helper.sh b/scripts/lvresize_fs_helper.sh
index 031b8453b..f531dd447 100755
--- a/scripts/lvresize_fs_helper.sh
+++ b/scripts/lvresize_fs_helper.sh
@@ -224,7 +224,7 @@ fsreduce() {
 cryptresize() {
 	NEWSIZESECTORS=$(($NEWSIZEBYTES/512))
 	logmsg "cryptsetup resize ${NEWSIZESECTORS} sectors ${DEVPATH}"
-	cryptresize resize --size "$NEWSIZESECTORS" "$DEVPATH"
+	cryptsetup resize --size "$NEWSIZESECTORS" "$DEVPATH"
 	if [ $? -eq 0 ]; then
 		logmsg "cryptsetup done"
 	else
-- 
2.39.0

